lib/liquid/expression.ex

```diff
@@ -4,7 +4,7 @@ defmodule Liquid.Expression do
 
   @literal_list [nil, "nil", "null", "", "true", "false", "blank", "empty"]
   @literals %{
-    nil: nil, "nil": nil, "null": nil, "": nil,
+    :nil=> nil, "nil": nil, "null": nil, "": nil,
```

Comment:
My IDEs were angry at using `nil` as a key, so I described it as an atom instead :)